src/pages/Quiz/QuizPage.jsx

```diff
@@ -0,0 +1,120 @@
+import { useState } from "react";
+import PageLayout from "../../components/PageLayout";
+import Button from "@mui/material/Button";
+import LinearProgress from '@mui/material/LinearProgress';
+import { v4 as uuidv4 } from "uuid";
+import { Box } from "@mui/system";
+
+/**
+ * Renders the Assessment page.
+ * @returns {JSX.Element} QuizPage component
+ */
+
+function QuizPage() {
+
+    const [currentQuestion, setcurrentQuestion] = useState(0);
+    const [progressValue, setprogressValue] = useState(1);
+
+    const handleAnswerButtonClick = (answer) => {
+        if (quiztions[currentQuestion].correct === answer.target.textContent) // checks if the answer is correct
+            console.log("Correct");
+        else
+            console.log("Incorrect");
+
+        if (currentQuestion !== quiztions.length - 1) { // updates questions and progress bar
+            setcurrentQuestion(currentQuestion + 1);
+            setprogressValue((((currentQuestion +1)/ quiztions.length) * 100));
+        }
+
+        // update the progress bar on the last question
+        if (currentQuestion === quiztions.length-1)
+            setprogressValue(100); 
+    }
+
+    const quiztions = [ // placeholder for the real questions
```

Comment:
What are quiztions? Do you mean questions?
Maybe you should name it dummyQuestions

Reply:
I thought it was a funny wordplay. It is changed now

Reply:
Please don't use wordplays in coding. You variables should be self-explanatory and nobody should guess what you mean with it.